### DISCUSSION TITLE
I can't newt connect as a device

### DISCUSSION THREAD

**Discussion by stoneblock** [Phambili-Tech/Newt_Display#69] [Q&A]
I've looked around because this error must be known, but I can't find anything.
I have a MacBookPro2014 running Mojave (OS10.14.6) 
The Newt does not mount as a device - after much testing it runs not directly in the programed system 
and in a emulated win8 it can be seen (but no known driver) - but this way it looks it's in the right mode.

Actually I can't reach the programming path for it or update the firmware to Newt-1.1.4-RC1.
Is there a driver to install on the old Mac system for the newt?
(in win 10 the esp32s2 is clearly recognised, but this system it's manged of my firm so i can't reach it there - the driver is not installed)

I found the cp210x is missing (part of the esp32 S2) - so I installed 
SiLabsUSBDriver.kext        - VCP Driver 6.0.2 
FTDIUSBSerialDriver.kext  - version 2.4.4 
both driver are now installed in (unchanged default path) /Library/Extensions/ 
at the normally place of older installations are no dead dublicates: /System/Library/Extensions/

cd /usr/local/lib
rm libftd2xx.1.4.4.dylib
rm libftd2xx.1.4.22.dylib
cp .../release/build/libftd2xx.1.4.30.dylib

cd /usr/local/lib
cp .../release/ftd2xx.h ftd2xx.h 
cp .../release/WinTypes.h WinTypes.h 

but the device is still hidden at mac os
any Idea

Thx

**Comment by darianbjohnson** [comment]
The ESP32s2 has native USB, so you shouldn't need to install an drivers. I didn't develop the Newt on a Mac... but let me see if I can try to duplicate the error with my kids Mac.

Also, you should be able to download the latest version directly from the Settings menu
